sql/core/src/main/scala/org/apache/spark/sql/execution/window/WindowExecBase.scala

```diff
@@ -136,8 +136,15 @@ trait WindowExecBase extends UnaryExecNode {
           val frame = spec.frameSpecification.asInstanceOf[SpecifiedWindowFrame]
           function match {
             case AggregateExpression(f, _, _, _, _) => collect("AGGREGATE", frame, e, f)
+            case f: OffsetWindowSpec if f.isRelative =>
```

Comment:
we should match `OffsetWindowFunction`, as it indicates `isRelative=true`, `ignoreNulls=false`, `frame= UnboundedPreceding and CurrentRow`.

Reply:
Good idea.